sdk/servicebus/Azure.Messaging.ServiceBus/tests/Diagnostics/EventSourceLiveTests.cs

```diff
@@ -105,6 +105,11 @@ await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitionin
                 await receiver.DisposeAsync();
                 _listener.SingleEventById(ServiceBusEventSource.ClientDisposeStartEvent, e => e.Payload.Contains(nameof(ServiceBusReceiver)) && e.Payload.Contains(receiver.Identifier));
                 _listener.SingleEventById(ServiceBusEventSource.ClientDisposeCompleteEvent, e => e.Payload.Contains(nameof(ServiceBusReceiver)) && e.Payload.Contains(receiver.Identifier));
+                // link closed event is fired asynchronously, so add a small
+                // delay
```

Comment:
nit: If the lines of code around the comment are wider than the comment, you don't need to split the comment into multiple lines. Generally, it is best to have separate comment lines to represent complete thoughts; unless the thought is very detailed/in-depth.

Reply:
Yeah, this was a mess up on my part.